lib/block_view.go

```diff
@@ -1182,7 +1188,8 @@ func (bav *UtxoView) _verifySignature(txn *MsgDeSoTxn, blockHeight uint32) (_der
 		}
 
 		// All checks passed so we try to verify the signature.
-		if txn.Signature.Verify(txHash[:], derivedPk) {
+		// FIXME: This step can be avoided for DeSo-DER signatures. At this point, we've already recovered and verified the signature.
+		if txn.Signature.Sign.Verify(txHash[:], derivedPk) {
```

Comment:
Let's resolve the FIXME. I think skipping this is as simple as checking if `IsRecoverable` is false before this and returning if so

Reply:
I'm always +1 on performance optimizations